less/index.less

```diff
@@ -50,10 +50,91 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
+	
 }
 
 html, body {
     height: 100%;
 }
 
+// Variables
+//@
+//@
+
+// Mixins
+.center() {
+	display: flex;
+	justify-content: center;
+	align-content: center;
+	
+}
+
+.font() {
+	font-size: 2rem;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+
+// General Styles
+h2 {
+	font-size: 1.5rem;
+}
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+  background-color: dodgerblue
+  
+}
```

Comment:
remember to nest everything inside the container block to show understanding of how nesting with preprocessing works/why it's so awesome -- just drop the ending `}` down the page to wrap around the rest of your styles!